README.md

```diff
@@ -464,6 +464,9 @@ network_params:
   # How long you want the network to wait before starting up
   genesis_delay: 20
 
+  # The gas limit of the network set at genesis
+  genesis_gas_limit: 30000000
```

Comment:
Could you please use `genesis_gaslimit` instead of `genesis_gas_limit`everywhere?

```suggestion
  genesis_gaslimit: 30000000
```
This would match the naming convention we have introduced earlier.